BookPlayer/Library/LibraryViewController.swift

```diff
@@ -58,15 +60,27 @@ class LibraryViewController: BaseListViewController, UIGestureRecognizerDelegate
         UIApplication.shared.endReceivingRemoteControlEvents()
     }
 
+    fileprivate func toggleEmptyStateView() {
```

Comment:
This declaration can go inside `BaseListViewController` since the implementation for `LibraryViewController` isn't different than the on in `PlaylistViewController`

Reply:
I wish. I can't add this to the parents as the outlets referenced do not exist in the parent and I haven't figured out how to override outlets in subclasses

Reply:
oh right 😅 my bad

Reply:
although there might be something, let me test something out

Reply:
alright:
1.- Move the `IBOutlet emptyStatePlaceholder` to `BaseListViewController` (delete the var from both Library and Playlist ViewControllers)
2.- From the storyboard reconnect both outlets from the two VCs into the one in the `BaseListViewController`
3.- Done

Reply:
Ohhh that works? 😅 Will fix asap.